ElementX/Sources/Services/Room/JoinedRoomProxy.swift

```diff
@@ -190,13 +190,32 @@ class JoinedRoomProxy: JoinedRoomProxyProtocol {
         }
     }
     
+    func threadTimeline(eventID: String) async -> Result<TimelineProxyProtocol, RoomProxyError> {
+        do {
+            let sdkTimeline = try await room.timelineWithConfiguration(configuration: .init(focus: .thread(rootEventId: eventID, numEvents: 20),
+                                                                                            filter: .all,
+                                                                                            internalIdPrefix: UUID().uuidString,
```

Comment:
Same as with the media timeline, we don't need this internal ID here as it is only to control the animation when switching the underlying timeline provider on the same screen.
```suggestion
                                                                                            internalIdPrefix: nil,
```